core/src/main/java/be/isach/ultracosmetics/listeners/PlayerListener.java

```diff
@@ -49,7 +49,7 @@ public PlayerListener(UltraCosmetics ultraCosmetics) {
         this.menuItem = ItemFactory.getMenuItem();
     }
 
-    @EventHandler(priority = EventPriority.LOWEST)
+    @EventHandler(priority = EventPriority.NORMAL)
```

Comment:
```suggestion
    @EventHandler
```
I'd prefer to leave off the event priority field here, since `NORMAL` is the default, that way it's easier to skim over the event handlers and see which ones priority matters for.